hack/install-kind.sh

```diff
@@ -1,40 +0,0 @@
-#!/bin/bash
```

Comment:
I personally would not remove those files as these are the only resources for developers to locally setup the respective resources to work on features or issues. For me, I find it much more appealing to have a project owned simple install script like this one rather than a somewhat generic to the quick start guide of another project. This increases the entry barrier. With these scripts, we are also in control of the respective versions.